clustered/ehcache-clustered/src/assemble/docker/server/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM azul/zulu-openjdk-alpine:17-jre-latest
+
+RUN apk update && apk upgrade && apk add bash
+RUN mkdir /terracotta && chgrp -R 0 /terracotta && chmod -R g+rX /terracotta
+RUN mkdir /terracotta/config && chgrp -R 0 /terracotta/config && chmod -R g+rX /terracotta/config
+RUN mkdir /terracotta/run && chgrp -R 0 /terracotta/run && chmod -R g+rwX /terracotta/run
+
+WORKDIR /terracotta
+
+# This volume can be mounted with "ro" flag.
+# This is where the configurations will be sourced and eventually copied elsewhere within the container.
+VOLUME "/terracotta/config"
+
+# This folder can be mounted with "rw" flag.
+# This is where the default configurations will output any logs, data, etc
+# If the volume is mounted, the container will support being restarted in the same state.
+# If a volume is mounted, be sure to set the correct chmod permissions in the host machine.
+# Also, you might need to use a special cleanup task to remove the created files in the host machine
+VOLUME "/terracotta/run"
+
+EXPOSE 9410
+
+ENV DEFAULT_OFFHEAP "offheap-1:512MB,offheap-2:512MB"
```

Comment:
Any reason we need two offheap resources by default in an Ehcache setup?

Reply:
I replicated the enterprise defaults. I prefer to keep them as-is because config tool and sever defaults also have 2 offheaps I think... Better to be consistent across all our maintained images.